jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/DataSetDefsBootstrap.java

```diff
@@ -61,13 +61,19 @@ protected void registerDataSetDefinitions() {
                             "log.correlationKey, " +
                             "log.externalId, " +
                             "log.processInstanceDescription, " +
-                            "COALESCE(info.lastModificationDate,log.end_date) as lastModificationDate " +
-                        "from " +
-                            "ProcessInstanceLog log " +
-                        "left join " +
-                            "ProcessInstanceInfo info " +
-                        "on " +
-                            "info.InstanceId=log.processInstanceId"
+                            "COALESCE(info.lastModificationDate,log.end_date) as lastModificationDate, " +
+                            "COALESCE(" +
```

Comment:
@bsremac we want to make sure this works fine with all supported DBs. Please make sure to implement & run some  some selenium tests with this in DB matrix.